### PR TITLE
missing information for redirect

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/10-Outlook/6-dispatching.rst [hunk 1]
```diff
@@ -283,16 +286,30 @@ returning a `ForwardResponse`::
...
+   
    $this->redirectToUri('https://example.com');
 
+.. note::
+
+   A redirection leads to a reload of the page. All the $_REQUEST variable is lost. Therefore all data needed on the 
+   destination must be passed as an array in parameter 4. 
+   If the redirection happens after a new / create form, then it mudst be taken care that the database record is stored
+   into the database before the redirection:
+   `$this->objectManager->get(\TYPO3\CMS\Extbase\Persistence\Generic\PersistenceManager::class)->persistAll();`
```
Comment (susannemoog) [inline Documentation/10-Outlook/6-dispatching.rst:303]: the object manager is deprecated and should not be used anymore - we should not add it to new documentation

Comment (franzholz) [reply Documentation/10-Outlook/6-dispatching.rst:303]: The ObjectManager has been deprecated in TYPO3 11.2. But this is no published TYPO3 yet. And it still works in TYPO3 11.2.
I have added a TODO.

Comment (susannemoog) [reply Documentation/10-Outlook/6-dispatching.rst:303]: Your PR targets "master" which is TYPO3 v11-dev and 11.2 has actually been released and is a published version, that's why we should not use the object manager here.

Comment (franzholz) [reply Documentation/10-Outlook/6-dispatching.rst:303]: But this Object Manager part is needed for the document versions for TYPO3 9 and 10.

Comment (linawolf) [reply Documentation/10-Outlook/6-dispatching.rst:303]: @franzholz please rebase your PR to LTS 10 if it is needed for that version and below only

Comment (franzholz) [reply Documentation/10-Outlook/6-dispatching.rst:303]: Most changes are needed also in TYPO3 11. 
Only a few lines are still working, but already outdated in TYPO3 11.

**Review by linawolf — APPROVED** [review]

**Pull request merged**